### PR TITLE
HTML to md

### PR DESCRIPTION
**Pull request by lbarraga**
In this PR, I converted nearly all the HTML in the `.md` files to plain Markdown.

**Initial State:**  
A project-wide search initially identified around 1,100 HTML tags across the files.

**Current State:**  
This number is now close to zero, with a few exceptions where conversion would significantly degrade the visuals.

**Key Changes:**
- The majority of the tags converted were `<code>`, `<b>`, and `<center>`.  
  - **Code and b Tags:** Commands are no longer in bold.
  - **Center Tags:** Images are no longer centered. I used MkDocs’ script title functionality instead of centered titles.
- **Tables:** I manually converted some large tables.

**Process:**
- I used regex for recurring patterns, but still manually reviewed each substitution. I then compared the before and after of the generated docs.

### CODE REVIEW

**Comment by lbarraga** [comment]
negative side effects: 

- commands in code blocks are no longer bold: 
image
vs 
image

- Tables cannot be styled less (see title text not centered):
image
vs
image

**Comment by stdweird** [comment]
what in the diff causes the table layout changes?

**Comment by boegel** [comment]
> what to do with these kinds of comments?
> 
> ```
> 
> ```
> 
> I have came across other weird "prompt" comments

Just remove them, we still have them in Git history?

**Comment by boegel** [comment]
> what in the diff causes the table layout changes?

Not using `<center>` HTML tags anymore, probably?

**Comment by stdweird** [comment]
i mean in this diff? i can't find the changes.

**Comment by stdweird** [comment]
i am merely wondering if the alignment works across multiple dividers. mkdocs needs an extension, not sure we already have it https://squidfunk.github.io/mkdocs-material/reference/data-tables/#configuration

so to try

```
| col1 | col 2 |
| :---:   | :---:    |
| centered | centered |
| :--- | ---: |
| left |  right |
```

**Comment by lbarraga** [comment]
> i am merely wondering if the alignment works across multiple dividers. mkdocs needs an extension, not sure we already have it https://squidfunk.github.io/mkdocs-material/reference/data-tables/#configuration
> 
> so to try
> 
> ```
> | col1 | col 2 |
> | :---:   | :---:    |
> | centered | centered |
> | :--- | ---: |
> | left |  right |
> ```

We are able to use this without loading an extra extension. The problem is that this centers the text in the cell, while the original html centers the text over the whole row. I have updated the image so the subtitle is in the second row (maybe this looks a bit more pleasing to the eye).

Another solution might also be to split this table into two tables with normal md titles for "long term storage" and "fast temporary storage"

**Comment by lbarraga** [comment]
> i mean in this diff? i can't find the changes.

That is my bad, I did not commit this yet. See next commit

**Comment by lbarraga** [comment]
I used the title feature for code blocks to get rid of the center tags before a codeblock:

image

vs

image


\```C title="T_hello.c"
....
\```

**Comment by lbarraga** [comment]
Images are no longer centered (left vs right):

image

**Comment by lbarraga** [comment]
In the original docs, HTML line breaks were used (`<br/>`) to render the last row of the following table:

image

This cannot be replicated in plain markdown. So I solved it like this:

image

**Comment by lbarraga** [comment]
This tables contained wrongly rendered `\newline`:
image

It is now:

image

**Comment by lbarraga** [comment]
`gpu.md` contains multiple html TODO comments. Should these be kept, removed or converted to issues?

**Comment by boegel** [comment]
> `gpu.md` contains multiple html TODO comments. Should these be kept, removed or converted to issues?

If you can't keep them easily commented out, then remove them and open issues for them

**Comment by lbarraga** [comment]
Besides HTML tags, I also see some images styled like this:

```
![image](img/ch0-hpc.png){: align=left style="height:383px;width:378px;border-radius:6px" }
```

I will not touch this as this seems deliberate.

File: mkdocs/docs/HPC/running_interactive_jobs.md [hunk 1]
```diff
@@ -380,15 +370,14 @@ screen, but also asks you to click a button.
 
 Now run the message program:
 
-<pre><code><b>$ cd ~/{{ exampledir }}</b>
-<b>./message.py</b>
-</code></pre>
+```
+$ cd ~/{{ exampledir }}
```
Comment (boegel) [inline mkdocs/docs/HPC/running_interactive_jobs.md:374]:
```suggestion
cd ~/{{ exampledir }}

File: mkdocs/docs/HPC/running_jobs_with_input_output_data.md [hunk 2]
```diff
@@ -13,18 +13,22 @@ and where that you can collect your results.
 
 First go to the directory:
 
-<pre><code><b>$ cd ~/{{ exampledir }}</b>
-</code></pre>
+```
+$ cd ~/{{ exampledir }}
```
Comment (boegel) [inline mkdocs/docs/HPC/running_jobs_with_input_output_data.md:17]:
```suggestion
cd ~/{{ exampledir }}

File: mkdocs/docs/HPC/running_jobs_with_input_output_data.md [hunk 3]
```diff
@@ -396,48 +325,54 @@ work.
 If you want to use the UGent shares longer than 24 hours, you should ask
 a ticket for up to a week by running
 
-<pre><code><b>$ kinit yourugentusername@UGENT.BE -r 7d</b>
-</code></pre>
+```
+$ kinit yourugentusername@UGENT.BE -r 7d
```
Comment (boegel) [inline mkdocs/docs/HPC/running_jobs_with_input_output_data.md:329]:
```suggestion
kinit yourugentusername@UGENT.BE -r 7

File: mkdocs/docs/HPC/running_jobs_with_input_output_data.md [hunk 4]
```diff
@@ -396,48 +325,54 @@ work.
...
 
 Your ticket is valid for 10 hours, but you can renew it before it
 expires.
 
 To renew your tickets, simply run
 
-<pre><code><b>$ kinit -R</b>
-</code></pre>
+```
+$ kinit -R
```
Comment (boegel) [inline mkdocs/docs/HPC/running_jobs_with_input_output_data.md:350]:
```suggestion
kinit -R

File: mkdocs/docs/HPC/running_jobs_with_input_output_data.md [hunk 5]
```diff
@@ -396,48 +325,54 @@ work.
...
+```
 
 If you want your ticket to be renewed automatically up to the maximum
 expiry date, you can run
 
-<pre><code><b>$ krenew -b -K 60</b>
-</code></pre>
+```
+$ krenew -b -K 60
```
Comment (boegel) [inline mkdocs/docs/HPC/running_jobs_with_input_output_data.md:357]:
```suggestion
krenew -b -K 60

File: mkdocs/docs/HPC/sites/gent/available-modules.md [hunk 6]
```diff
@@ -1,21 +1,23 @@
...
    ...
-</code></pre>
+```
 
 Or when you want to check whether some specific software, some compiler or some
 application (e.g., MATLAB) is installed on the {{hpc}}.
 
-<pre><code>$ <b>module av matlab</b>
+```
+$ module av matlab
```
Comment (boegel) [inline mkdocs/docs/HPC/sites/gent/available-modules.md:16]:
```suggestion
module avail matlab

File: mkdocs/docs/HPC/troubleshooting.md [hunk 7]
```diff
@@ -389,14 +370,16 @@ If it doesn't, or you are in doubt, take a screenshot, press "Yes" and contact {
 
 If you get errors like:
 
-<pre><code><b>qsub fibo.pbs</b>
+```
+qsub fibo.pbs
```
Comment (boegel) [inline mkdocs/docs/HPC/troubleshooting.md:374]:
```suggestion
$ qsub fibo.pbs

File: mkdocs/docs/HPC/troubleshooting.md [hunk 8]
```diff
@@ -405,17 +388,20 @@ See the [section about `dos2unix` in Linux tutorial](../linux-tutorial/uploading
 
 {% if OS == (linux or macos) %}
 
-<pre><code><b>ssh {{userid}}@{{loginnode}}</b>
-The authenticity of host {{loginnode}} (&lt;IP-adress&gt;) can't be established. 
-<u>&lt;algorithm&gt; key fingerprint is &lt;hash&gt;</u>
+```
+ssh {{userid}}@{{loginnode}}
```
Comment (boegel) [inline mkdocs/docs/HPC/troubleshooting.md:392]:
```suggestion
$ ssh {{userid}}@{{loginnode}}

File: mkdocs/docs/HPC/useful_linux_commands.md [hunk 9]
```diff
@@ -84,10 +43,11 @@ explained then if necessary. If not, you can usually get more
...
 either of the following:
 
-<pre><code>$ <b>ls --help </b>
-$ <b>man ls</b>
-$ <b> info ls </b>
-</code></pre>
+```
+$ ls --help 
+$ man ls
+$  info ls
```
Comment (boegel) [inline mkdocs/docs/HPC/useful_linux_commands.md:49]:
```suggestion
$ info ls

File: mkdocs/docs/HPC/useful_linux_commands.md [hunk 10]
```diff
@@ -125,38 +85,43 @@ hostname
...
+```
 
 Suppose we want to call this script "foo". You open a new file for
 editing, and name it "foo", and edit it with your favourite editor
 
 {% if site == gent %}
-<pre><code>$ <b>nano foo</b>
-</code></pre>
+```
+$ nano foo
```
Comment (boegel) [inline mkdocs/docs/HPC/useful_linux_commands.md:100]:
```suggestion
nano foo

File: mkdocs/docs/HPC/useful_linux_commands.md [hunk 11]
```diff
@@ -188,15 +154,17 @@ script.
 Finally, we tell the operating system that this script is now
 executable. For this we change its file attributes:
 
-<pre><code>$ <b> chmod +x foo</b>
-</code></pre>
+```
+$  chmod +x foo
```
Comment (boegel) [inline mkdocs/docs/HPC/useful_linux_commands.md:158]:
```suggestion
chmod +x foo

File: mkdocs/docs/HPC/x2go.md [hunk 12]
```diff
@@ -175,8 +163,9 @@ session, you should know which login node were used at first place. You
 can get this information before logging out from your X2Go session. Just
 open a terminal and execute:
 
-<pre><code>$ <b>hostname</b>
-</code></pre>
+```
+$ hostname
```
Comment (boegel) [inline mkdocs/docs/HPC/x2go.md:167]:
```suggestion
hostname

**Review by boegel — CHANGES_REQUESTED** [review]
I've gone through this in details, looks great, thanks a lot for the effort on this one @lbarraga!

I've made a large bunch of suggestions for tiny edits, which I'll apply myself.

They pretty much all consist of removing `$ ` in code blocks which only specify a command to run,  which makes them a lot easier to copy-paste, so I took the opportunity to improve that a lot while reviewing this...

**Review by boegel — APPROVED** [review]

**Pull request merged**